### PR TITLE
Update DOCS.md

### PR DESCRIPTION
**Pull request by TheDK**
Fix typo…

# Proposed Changes

> (Describe the changes and rationale behind them)

## Related Issues

> ([Github link][autolink-references] to related issues or pull requests)

[autolink-references]: https://help.github.com/articles/autolinked-references-and-urls/

### CODE REVIEW

File: zwave-js-ui/DOCS.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Home Assistant Community Add-on: Z-Wave JS UI
 
 The Z-Wave JS UI add-on provides an additional control panel, allowing you
-to configure every aspect of your Z-Wave network. It provides a decouled
+to configure every aspect of your Z-Wave network. It provides a decoupled
```
Comment (frenck) [inline zwave-js-ui/DOCS.md:4]: This same typo is in the README and the README template (as the same text is used). Could you correct/keep them in sync?

../Frenck

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by TheDK** [comment]
Done

**Comment by TheDK** [comment]
@frenck I corrected both docs - please feel free to merge.. :)

**Comment by frenck** [comment]
@TheDK I'm aware, started the maintenance round on this add-on yesterday as well.. didn't get to complete the round yet, as weekend/family life got in the way.

**Comment by TheDK** [comment]
No worries, just countering the stale bot in case you missed it…

**Review by frenck — APPROVED** [review]
Thanks, @TheDK 👍

**Pull request merged**